tests/CodeGenerators/SettingsGen.UnitTests/FileGeneration/XmlFileGenerationUnitTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.CodeGenerators.SettingsGen.FileGeneration;
+using Microsoft.Omex.CodeGenerators.SettingsGen.Models;
+using Microsoft.Omex.System.UnitTests.Shared;
+using Xunit;
+
+namespace Microsoft.Omex.CodeGenerators.SettingsGen.UnitTests.FileGeneration
+{
+	public sealed class XmlFileGenerationUnitTests : UnitTestBase
+	{
+
+		[Fact]
```

Comment:
Not sure how it's relevant, but some time ago we were pushing change from XUnit to MSTest

Reply:
Think I will keep it as xunit for now